resources/views/external-link-confirm.blade.php

```diff
@@ -6,61 +6,81 @@
 
 @php($identifier = md5($url . '-' . Str::random(8)))
 
-<a
-    href="javascript:;"
-    x-data="{
+<div x-data="{
         openModal() {
             Livewire.emit('openModal', '{{ $identifier }}')
+        },
+        redirect() {
+            window.open('{{ $url }}', '_blank')
+        },
+        hasConfirmedLinkWarning() {
+            return localStorage.getItem('confirm_link_warning') == 'true';
+        },
+        hasDisabledLinkWarning() {
+            return localStorage.getItem('disable_link_warning') == 'true';
         }
     }"
-    class="inline-flex items-center space-x-2 font-semibold whitespace-nowrap cursor-pointer link"
-    @click="openModal"
 >
-    <span>{{ $text ?? $slot ?? '' }}</span>
+    <a
+        href="javascript:;"
+        class="inline-flex items-center space-x-2 font-semibold whitespace-nowrap cursor-pointer link"
+        @click="hasDisabledLinkWarning() ? redirect() : openModal()"
+    >
+        <span>{{ $text ?? $slot ?? '' }}</span>
 
-    @unless($noIcon)
-        <x-ark-icon name="link" size="sm" class="mx-2" />
-    @endunless
-</a>
+        @unless($noIcon)
+            <x-ark-icon name="link" size="sm" class="mx-2" />
+        @endunless
+    </a>
 
-@push('footer')
-    <x-ark-js-modal
-        :name="$identifier"
-        class="w-full max-w-2xl text-left rounded-xl"
-        title-class="header-2"
-        buttons-style="flex justify-end space-x-3"
-        init
-    >
-        @slot('title')
-            @lang('generic.external_link')
-        @endslot
+    @push('footer')
+        <x-ark-js-modal
+            :name="$identifier"
+            class="w-full max-w-2xl text-left rounded-xl"
+            title-class="header-2"
+            buttons-style="flex justify-end space-x-3"
+            init
+        >
+            @slot('title')
+                @lang('generic.external_link')
+            @endslot
+
+            @slot('description')
+                <div class="flex flex-col mt-8 space-y-4 whitespace-normal">
+                    <div class="font-semibold text-theme-secondary-900">
+                        <x-ark-alert type="warning" :message="$url" />
+                    </div>
+                    <p>@lang('generic.external_link_disclaimer')</p>
 
-        @slot('description')
-            <div class="flex flex-col mt-8 space-y-4 whitespace-normal">
-                <div class="font-semibold text-theme-secondary-900">
-                    <x-ark-alert type="warning" :message="$url" />
+                    <x-ark-checkbox
+                        name="confirmation"
+                        alpine="localStorage.getItem('confirm_link_warning') == 'true' ? localStorage.setItem('confirm_link_warning', false) : localStorage.setItem('confirm_link_warning', true)"
```

Comment:
maybe better name is `has_confirmed_link_warning` so it's clear that it was about that and not some confirmation?